src/mono/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.iOS.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
+  <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Runtime.CompilerServices.RuntimeFeature">
+      <method signature="System.Boolean get_IsDynamicCodeCompiled()" body="stub" value="false" />
```

Comment:
Why do it this way? Why not have an #if in `RuntimeFeature.Mono.cs` for iOS and just hard code `false` in the code?